### PR TITLE
Flow format reference regeneration

### PR DESCRIPTION
**Pull request by skrthomas**
Version(s):

Merge to only the no-1.6 branch - no cherrypicks are required.
This PR is part of an experiment for simplifying merges for asynchronous content, and I will open one PR against main to incorporate all of the Network Observability 1.6 content just before its GA
Issue:

Link to docs preview:

https://76599--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/network_observability/json-flows-format-reference.html
QE review:
- [X] QE has approved this change.

Additional information:

### CODE REVIEW

File: _topic_maps/_topic_map.yml [hunk 1]
```diff
@@ -2692,7 +2692,7 @@ Topics:
     File: network-observability-operator-monitoring
   - Name: API reference
     File: flowcollector-api
-  - Name: JSON flows format reference
+  - Name: Flows format reference
```
Comment (skrthomas) [inline _topic_maps/_topic_map.yml:2695]: @jotak as you requested, I updated the title to remove "JSON"

**Comment by skrthomas** [comment]
@jotak @memodi can you PTAL at this flows format reference regeneration?

**Review by memodi — APPROVED** [review]
/label qe-approved

**Review by jotak — APPROVED** [review]

**Comment by aldiazRH** [comment]
LGTM!

**Comment by bscott-rh** [comment]
Merge review LGTM.

**Pull request merged**